### PR TITLE
point docs to core docs site (Close #119)

### LINKED ISSUE #119: Migrate core docs to snowplow.github.io Public
### Describe the feature
Move the main package [docs](https://github.com/snowplow/dbt-snowplow-web/blob/main/docs/markdown/snowplow_web_overview.md) into the core snowplow docs site to enable improved navigation and search, as well as prepare for increase size of test section for #116

### PR DESCRIPTION
**Pull request by rlh1994**
## Description & motivation
Remove most of the docs that are now on core docs and point to core docs site.

DO NOT MERGE until https://github.com/snowplow/documentation/pull/113 has been closed 

## Checklist
- [x] I have verified that these changes work locally
- [x] I have updated the README.md (if applicable)
- [x] I have added tests & descriptions to my models (and macros if applicable)

### CODE REVIEW

**Comment by snowplowcla** [comment]
Thanks for your pull request. Is this your first contribution to a Snowplow open source project? Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

:memo: Please visit https://docs.snowplowanalytics.com/docs/contributing/contributor-license-agreement/ to learn more and sign.

Once you've signed, please reply here (e.g. I signed it!) and we'll verify. Thanks.

**Review by paulboocock — APPROVED** [review]

File: docs/markdown/snowplow_web_overview.md [hunk 1]
```diff
@@ -4,9 +4,11 @@
...
+Welcome to the model documentation site for the Snowplow web dbt package. The package contains is a fully incremental model, that transforms raw web event data generated by the [Snowplow JavaScript tracker](https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/javascript-trackers/) into a series of derived tables of varying levels of aggregation.
 
-**Note this doc site is linked to latest release of the package. If you are not using the latest release, [generate and serve](https://docs.getdbt.com/reference/commands/cmd-docs#dbt-docs-serve) the doc site locally for accurate documentation.**
+**For more information, including a QuickStart guide, operation and configuration, and implementing your own custom modules on top of this please visit the [Snowplow Docs](https://docs.snowplowanalytics.com/docs/modeling-your-data/modeling-your-data-with-dbt/).**
```
Comment (paulboocock) [inline docs/markdown/snowplow_web_overview.md:9]:
```suggestion
**For more information, including a QuickStart guide, operation and configuration, and implementing your own custom modules on top of this please visit the [Snowplow Docs](https://docs.snowplow.io/docs/modeling-your-data/modeling-your-data-with-dbt/).**
```

**Review by paulboocock — CHANGES_REQUESTED** [review]
Changed my mind ;-)

File: docs/markdown/snowplow_web_overview.md (hunk 1 above)
Comment (rlh1994) [reply docs/markdown/snowplow_web_overview.md]: Replaced all instances to snowplow.io

**Review by paulboocock — APPROVED** [review]

**Pull request merged**